css-three-column-layout/styles.css

```diff
@@ -0,0 +1,51 @@
+body {
+  justify-content: center;
+}
+
+.container {
+  background-color: rgb(67, 129, 229);
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  font-family: lato, sans-serif;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  gap: 2rem;
+}
```

Comment:
In the future I might recommend setting this as a just a flex property and then adding the other items to another class. This way you can re-use `.row` without always having a gap and justify content.
```
.row {
  display: flex;
}

.just-center-gap {
  justify-content: center;
  gap: 2rem;
}
```

You don't have to use that name, it's just something random I made since it describes using justify-content and gap.